avocado/core/sysinfo.py

```diff
@@ -491,30 +124,41 @@ def _get_syslog_watcher():
                     "/var/log/system.log"]
         for logpath in logpaths:
             if os.path.exists(logpath):
-                return LogWatcher(logpath)
+                try:
+                    return sysinfo.LogWatcher(logpath)
+                except PermissionError as e:
+                    log.debug(e.args[0])
         raise ValueError("System log file not found (looked for %s)" %
                          logpaths)
 
     def _set_collectibles(self):
+        timeout = self.config.get('sysinfo.collect.commands_timeout', -1)
+        locale = self.config.get('sysinfo.collect.locale', 'C')
```

Comment:
@beraldoleal, did you mean @richtja should add the default values here or not add them and let it use what was initialized?

I ask because if someday we change the default value, we lose the ability to change it automatically, and we need to scan the code for places where it is setting a default value.

Reply:
I mean to not use the default value at all here, unless we have a good reason for it:

```python
 timeout = self.config.get('sysinfo.collect.commands_timeout')
locale = self.config.get('sysinfo.collect.locale')
```

This should be returning already the default values.

Reply:
I had problems getting the default values for some config options and had to set it, but if we can use the default from configs, that should be the preferred way, in my opinion.

Reply:
Yes. Sorry if I wasn't clear @richtja . What I mean is: If we can't use the default values provided at registration time, probably is a very special case and we should investigate to understand.

Are you able to get the default values here?

```python
timeout = self.config.get('sysinfo.collect.commands_timeout')
locale = self.config.get('sysinfo.collect.locale')
```

Reply:
Yes, I am able to get default values here, I will change that.